Contentful.Core/ContentfulClient.cs

```diff
@@ -299,6 +304,11 @@ private void ResolveLinks(JObject json, JObject entryToken, ISet<string> process
 
                 var grandParent = (JObject)linkToken.Parent.Parent;
 
+                if((grandParent.Parent.Parent.Parent.Parent is JObject) && grandParent.Parent.Parent.Parent.Parent.Value<string>("nodeType") == "embedded-entry-block")
```

Comment:
😵

Reply:
Yeah, this is what happens with uneccessary nesting... data.target.sys